### PR TITLE
FAQ.rst: Shorten "What does coala do(for me)?"

### PR DESCRIPTION
**Pull request by ujdcodr**
This answer removes the analogy to LibreOffice.
coala's key features that previously created
clutter in the main paragraph, have been added
to the bullet points instead.
It emphasizes the two important points, making
it clear to users about what coala is and what
it does

Closes https://github.com/coala/documentation/issues/289

### CODE REVIEW

**Comment by pratyushprakash** [comment]
ack 2efda9eadcc8cc9de7cac72ecac80c4a33d1780a

**Comment by aptrishu** [comment]
unack 2efda9eadcc8cc9de7cac72ecac80c4a33d1780a

**Comment by aptrishu** [comment]
Use Imperative mood in commit shortlog.

**Comment by ujdcodr** [comment]
@aptrishu Is this fine?

File: Help/FAQ.rst [hunk 1]
```diff
@@ -7,24 +7,16 @@ questions by newcomers or even contributors.
...
-- Make it easy to use existing static code analyzers by unifying and
-  simplifying the configs
-- Make it easy to write new routines by offering the interface part
-  and everything but the actual analyzer routine.
+coala is like a spell and grammar checker for source code. It offers one
+command and one configuration to lint all languages in your project.
+
+In short coala does two things:
+
+- Make it easy to use existing static code analyzers such as clang,
```
Comment (RohanVB) [inline Help/FAQ.rst]: Makes* it easy since we are referring to what coala does.

File: Help/FAQ.rst [hunk 2]
```diff
@@ -7,24 +7,16 @@ questions by newcomers or even contributors.
...
+  pylint(and many others) by unifying and simplifying the configs
+- Make it easy to write new routines by offering the interface part and
```
Comment (RohanVB) [inline Help/FAQ.rst]: Makes* it easy since we are referring to what coala does.

**Comment by ujdcodr** [comment]
Ready for merge?

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/393/commits/eb540c12238c4ef0819b62fa52dfd83aeccd5758

**Comment by sils** [comment]
@rultor merge

**Comment by ujdcodr** [comment]
Thanks @sils . This is the first of many more PRs to come.

**Pull request merged**